### ISSUE TITLE
Support specifying python_version in app_config.yaml for all examples

### ISSUE THREAD

**Issue by carlosdtrejo** [enthought/edge-examples#103]
From support ticket [#164](https://edge-support.enthought.com/a/tickets/164?current_tab=details)

Currently, the python_version specified in app_config.yaml in all examples does not affect the Python version in the EDM environment, which defaults to 3.8. It would be useful to allow the specified python_version to be applied when building the image.

Create issue in edge-product to update edge_cli accordingly.

**Comment by carlosdtrejo** [comment]
cc @dpinte @EdwinWiseOne

**Comment by dpinte** [comment]
@carlosdtrejo +1 that is very important, especially now that Python 3.8 is EOL

**Comment by EdwinWiseOne** [comment]
@dpinte We don't specify platform in the config, but are hard-coding to rh7, which when we use python 3.11 won't work, so I think updating to rh8 with this would be good (and rh8 should work with python 3.8 also).

**Comment by dpinte** [comment]
@EdwinWiseOne unfortunately, rh8 won't work with Python 3.8.

**Comment by EdwinWiseOne** [comment]
@dpinte Yeah I saw that and adjusted, but also Plotly Dash and Panel don't have their packages in 3.11.  So I left them at 3.8

**Comment by dpinte** [comment]
@EdwinWiseOne can you open a buildsystem request to get those?
